js- academy - GOL - first homework

hrdlim01 opened the pull request:
- finished GOL rules
- unit tests

hrdlim01 committed:
js- academy - GOL - first homework
- finished GOL rules
- unit tests

dominiktilp commented on src/gol.js:
```diff
@@ -1,3 +1,16 @@
+export function countLiveNeighbours(board, i, j) {
```
It is really a good practice to define function before you use it. ;-) (y)

dominiktilp commented on src/gol.js:
```diff
@@ -13,22 +26,15 @@ export default function next(input) {
       const liveNeighbours = countLiveNeighbours(input, i, j);
       if (liveNeighbours < 2) {
         output[i][j] = 0;
+      } else if (liveNeighbours === 3) {
```
This is as easy as possible :-)

dominiktilp commented:
I like this easy-peasy solution. It fulfil all requirements but is as minimal as possible.

hrdlim01 committed:
js- academy - GOL - React homework
- Components for setting the game board and its initial state,
- playing the game fluently,
- stopping the fluent game,
- running the game step by step,
- reseting the game,
- game generation counter

hrdlim01 committed:
js- academy - GOL - React homework
- added some missing files

hrdlim01 committed:
js- academy - GOL - React homework
- added some missing files
